app/assets/stylesheets/dashboard.scss

```
@@ -683,7 +683,8 @@
       }
 
       &.proposal-code {
-        font-size: 1.7cm;
+        font-size: 2cm;
```

Whether this style is the same as in the preview or not depends on your screen resolution and font size :thinking:. So, when in doubt, I'd vote to leave the original style as it was.

```suggestion
        font-size: 1.7cm;
```